### ISSUE TITLE
Community Team Member application: Mariatta

### ISSUE THREAD

**Issue by Mariatta** [coala/teams#22]
# Bio

My name is Mariatta, and I'm extra super duper special because:
- At the time of applying, I'm the only woman in coala maintainers team.
- I'm on IMDb http://www.imdb.com/name/nm7641957/

My other community involvements include:
- co-organize Vancouver PyLadies meetups
- organized a screening of Code: Debugging the Gender Gap with VanPy User group
- mentored for Girls Learning Code and Ladies Learning Code
- speaking at conferences: Vancouver Python Day 2016, DjangoCon US 2016, PyCaribbean 2017
- attend local Python meetups

In essence, I've been part of the community already, you might as well just accept me 😜 

# coala Contributions so far

- documentation
- review prs
- accept prs
- helped onboard newcomers and new contributors

# Road to the Future

- help improve newcomer's experience so that they'll stay and continue contributing
- help improve documentation
- help promote coala by giving lightning talks
- suggest that we adopt a Code of Conduct, similar to Python's or Django's CoC. 
- be on the lookout for upcoming conferences for coala to make a presence

**Comment by sims1253** [comment]
looking at the [list so far](https://github.com/coala/teams/blob/master/community/community_ideas.md)
What are areas/specific tasks you would like to be responsible for?
Are there things you would like to add to that list? Besides the doc part (which belongs to the doc team) I feel like your points are covered by the list :)

**Comment by sims1253** [comment]
Also you applied to two other teams. Do you feel that you can dedicate time and focus on three teams/areas?

**Comment by sils** [comment]
@Mariatta according to your imdb profile you're good with graphics and that sounds hell impressive! Do you think we can do something like a great video about coala similar to [what GNOME does](https://www.youtube.com/watch?v=LhY7rpWXm1Y) since a year or so? I think creating some great visual content to attract users and developers alike can help us a lot, we're also looking towards a new video for the frontpage.

Though those things are a bit more technical I think they do relate to the community aspects. Would you like to help us here or have any ideas on how we can make it rock better :)?

**Comment by sils** [comment]
I hereby vote for this proposal

**Comment by Mariatta** [comment]
@sims1253 
> Also you applied to two other teams. Do you feel that you can dedicate time and focus on three teams/areas?

Being involved in Python community is something I'm passionate with, and it's also something I've been doing already, it won't take much more time and effort than what I've put in already.

When given new assignments, I'll disclose my prior commitments and whether I have other tasks on hand and let them know if there will be any scheduling conflict. I know we'll all be able to work things out.

Since there are other community team members here, I think you'll have enough backup bodies to pick up my slack lol

> Are there things you would like to add to that list?

My immediate goal will be to add a code of conduct to our community. I saw you added the entry to that list (Thanks!!!) Otherwise, I think our goals are aligning.
I'm already planning to give a lightning talk about coala at vancouver pyladies Jan 31.
I might consider giving a lighting talk about coala at PyCaribbean in February 😉 

@sils 
> you're good with graphics

Seems like bragging about imdb backfired on me. It says I'm known for visual effects but in fact I know nothing about visual effects lol. I'm a software engineer and I build the internal infrastructure for my company, which happens to be a visual effects company.
So in conclusion, I don't actually have the skills to help you create new video, but I have connections  (photographers and animators) I can try reaching out and see if they're interested to help out. (We might need to entice them with swag/chocolates/beer) 
I know this is not the news you're hoping to hear, but I'll do my best

**Comment by sils** [comment]
> I can try reaching out and see if they're interested to help out. (We might need to entice them with swag/chocolates/beer)

Let me know when you need the swag so we can arrange that ;)

**Comment by sims1253** [comment]
:tada: I vote for this two. I don't have time to create your inclusion PR (see the ones for Lasse and Hemang). I'd do it tomorrow I guess but feel free to do it yourself if you can't wait :)

**Comment by sils** [comment]
isn't this resolved?
